File: Php/app/Urls/CashgameDeleteUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\CashgameUrlModel;
	use entities\Cashgame;
	use entities\Homegame;

	class CashgameDeleteUrlModel extends CashgameUrlModel{

		public function __construct(Homegame $homegame, Cashgame $cashgame){
			parent::__construct(RouteFormats::cashgameDelete, $homegame, $cashgame);
		}

	}

}

File: Web/Models/UrlModels/CashgameDeleteUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class CashgameDeleteUrlModel : CashgameUrlModel{

		public CashgameDeleteUrlModel(Homegame homegame, Cashgame cashgame)
            : base(RouteFormats.CashgameDelete, homegame, cashgame)
        {
		}

	}

}